submissions/mustbefail/friends-app/src/js/app.js

```diff
@@ -0,0 +1,83 @@
+import { API_REQ, sortUsers, filterUsers, resetFilters } from './services.js';
+import makeCardTemplate from './template.js';
+
+const contactsContainer = document.querySelector('.contacts-container');
+
+const render = async (state) => {
+  const users = await getUsers(API_REQ);
+  const sortedUsers = sortUsers(state.sort, users);
+  const filteredUsers = filterUsers(state.filter, sortedUsers);
+  const cards = filteredUsers
+    .map((user) =>
+      makeCardTemplate(
+        user.name,
+        user.email,
+        user.location.city,
+        user.dob.date,
+        user.dob.age,
+        user.phone,
+        user.picture.large
+      )
+    )
+    .join('');
+  contactsContainer.innerHTML = cards;
+};
+
+const getUsers = async (url) => {
+  try {
```

Comment:
You can keep url in this function because this variable not used and never will be anywhere else..